test/SparqlJsonParser-test.ts

```diff
@@ -96,6 +96,17 @@ describe('SparqlJsonParser', () => {
 `)))).toEqual([]);
     });
 
+    it('should convert a slightly invalid empty SPARQL JSON response (common PHP error)', async () => {
```

Comment:
Why is this slightly invalid? (looks valid to me at first glance)

Reply:
Maybe because `bindings` is not an array as started in the specification?

Reply:
Because the `"bindings"` key value should be an array and not an object.